### PR TITLE
PIM-1449: PIM | AMS | Topic 2 | C | Numeric Import Attribute, Data & Export

### PR DESCRIPTION
**Pull request by KimJinPang**
Corresponding propel-document-java PR: https://github.com/PropelPLM/propel-document-java/pull/54

- Import numerical values for creation of products and variant values
- Import numerical values for updating of products and variant values
- Add min and max value support for numerical value import
- Export numerical values (from detail page, list page, xlsx, csv, no template, xlsx template, and csv template)
- Abstract out obtaining of `Attribute_Value__c`'s value into helper function

image

### CODE REVIEW

**Review by danielxji6 — APPROVED** [review]
⭐️⭐️⭐️

**Review by ailanthustng — APPROVED** [review]
left a few suggestions, otherwise looks good!

File: legacy/ExportPIM.js [hunk 1]
```diff
@@ -153,7 +153,11 @@ function convertArrayOfObjectsToCSV(
       ) {
         recordAttributes.set(skey, recordAttributes.get(skey).Name);
       }
-      csvStringResult += cleanString(records[i].get(skey) || '');
+      if (typeof records[i].get(skey) === 'number') {
```
Comment (ailanthustng) [inline legacy/ExportPIM.js:156]: do we want to put these values somewhere as a constant? same with other parts of the PR like 'Value__c', etc

File: legacy/PimStructure.js [hunk 2]
```diff
@@ -202,7 +201,7 @@ class PimStructure {
             if (valuesIdList.length > 0) {
               overwrittenValues = await service.simpleQuery(
                 helper.namespaceQuery(
-                  `select Id, Attribute_Label__c, Attribute_Label_Type__c, Value__c, Value_Long__c, Product__c, Overwritten_Variant_Value__c
+                  `select Id, Attribute_Label__c, Attribute_Label_Type__c, Value__c, Value_Long__c, Numeric_Value__c, Product__c, Overwritten_Variant_Value__c
```
Comment (ailanthustng) [inline legacy/PimStructure.js:204]: can perhaps put these into an array then join them. same for the other queries

**Review by ailanthustng — APPROVED** [review]

**Pull request merged**